.gitignore

```diff
@@ -21,6 +21,7 @@ opentelemetry-java-contrib-jmx-metrics.jar
 VERSION.txt
 release_deps
 /tmp
+tmp
```

Comment:
We already ignore temp can you condense this line and the one above with `**/tmp` I think that should ignore either path.